virttest/env_process.py

```diff
@@ -1479,6 +1488,9 @@ def postprocess(test, params, env):
 
     if err:
         raise RuntimeError("Failures occurred while postprocess:\n%s" % err)
+    elif _post_hugepages_surp > _pre_hugepages_surp:
+        leak_num = _post_hugepages_surp - _pre_hugepages_surp
+        raise exceptions.TestError("%d huge pages leaked!" % leak_num)
```

Comment:
It should be considered as TestFail right ?

Reply:
Yes, TestFail looks better than TestError, has been updated.